mod.config

```diff
@@ -119,25 +123,24 @@ AUTOMATIC_ENGINE_EXTRACT_DIRECTORY="./engine_temp"
 AUTOMATIC_ENGINE_TEMP_ARCHIVE_NAME="engine.zip"
 ENGINE_DIRECTORY="./engine"
 
-# The url to download the OpenRA macOS launcher files.
-PACKAGING_OSX_LAUNCHER_SOURCE="https://github.com/OpenRA/OpenRALauncherOSX/releases/download/${PACKAGING_OSX_LAUNCHER_TAG}/launcher.zip"
+# The url to download the OpenRA macOS mono runtime.
+PACKAGING_OSX_MONO_SOURCE="https://github.com/OpenRA/OpenRALauncherOSX/releases/download/${PACKAGING_OSX_MONO_TAG}/mono.zip"
 
 # Temporary file name used when downloading the OpenRA macOS launcher files.
-PACKAGING_OSX_LAUNCHER_TEMP_ARCHIVE_NAME="launcher.zip"
+PACKAGING_OSX_MONO_TEMP_ARCHIVE_NAME="mono.zip"
 
 # The url to download the OpenRA AppImage dependencies.
-PACKAGING_APPIMAGE_DEPENDENCIES_SOURCE="https://github.com/OpenRA/AppImageSupport/releases/download/${PACKAGING_APPIMAGE_DEPENDENCIES_TAG}/libs.tar.bz2"
+PACKAGING_APPIMAGE_DEPENDENCIES_SOURCE="https://github.com/OpenRA/AppImageSupport/releases/download/${PACKAGING_APPIMAGE_DEPENDENCIES_TAG}/mono.tar.bz2"
 
 # Temporary file name used when downloading the OpenRA AppImage dependencies.
-PACKAGING_APPIMAGE_DEPENDENCIES_TEMP_ARCHIVE_NAME="libs.tar.bz2"
+PACKAGING_APPIMAGE_DEPENDENCIES_TEMP_ARCHIVE_NAME="mono.tar.bz2"
 
 # List of .NET assemblies that we can guarantee exist
-# OpenRA.Game.dll is a harmless false positive that we can ignore
-WHITELISTED_OPENRA_ASSEMBLIES="OpenRA.Game.exe OpenRA.Utility.exe OpenRA.Platforms.Default.dll OpenRA.Mods.Common.dll OpenRA.Game.dll"
+WHITELISTED_OPENRA_ASSEMBLIES="OpenRA.exe OpenRA.Utility.exe OpenRA.Server.exe OpenRA.Platforms.Default.dll OpenRA.Game.dll OpenRA.Mods.Common.dll OpenRA.Mods.Cnc.dll OpenRA.Mods.D2k.dll"
```

Comment:
You set `PACKAGING_COPY_CNC_DLL="False"` and `PACKAGING_COPY_D2K_DLL="False"` so this looks inconsistent at first glance.

Reply:
This is a good point, but I think the immediate fix here would be even worse than the current problem. This whitelist is under the advanced config that we expect that most projects shouldn't touch, whereas `PACKAGING_COPY_*_DLL` is standard config.

Considering that this has already been an issue for the last couple of SDK releases, and these whitelist entries will be going away completely when the SDK switches to the next engine release with https://github.com/OpenRA/OpenRA/pull/18955, I believe that the safer option is to keep it as it is here.